tutorials/20-structured-data.ipynb

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    for _, row in table.iterrows():
</pre><p>Similarity directly use table for the output. Please also remove print in the function to avoid confusions. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/evadb/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>